tests/scanners/test.javascript.js

```diff
@@ -290,7 +290,7 @@ describe('JavaScript Scanner', function() {
   it('should export all rules in rules/javascript', () => {
     // We skip the "run" check here for now as that's handled by ESLint.
     var ruleFiles = getRuleFiles('javascript');
-    var externalRules = 1;
+    var externalRules = 2;
```

Comment:
At some point it would be good to look at updating the eslintrc of the codebase so that `const` and `let` are used (preferring const) and then all the var usage can be removed.